spec/address_geocoder_spec.rb

```diff
@@ -112,21 +112,32 @@
         # when only street
         address_geocoder = AddressGeocoder.new({api_key: ENV['AddressGeocoderApiKey'], country: 'CH', street: '10, On Lok Mun Street'})
         expect(address_geocoder.suggested_addresses).to eq(false)
+        # when correct city vs wrong state
+        address_geocoder = AddressGeocoder.new({api_key: ENV['AddressGeocoderApiKey'], country: 'CN', city: 'Hangzhou', state: 'Osaka Prefecture'})
```

Comment:
because it depends on Google MAP API
even though i can recognise, it will still return false when google says `ZeroResult`.